corehq/apps/analytics/tasks.py

```diff
@@ -247,7 +247,7 @@ def update_hubspot_properties(webuser, properties):
     update_hubspot_properties_v2(webuser, properties)
 
 
-@analytics_task()
+@analytics_task(serializer='pickle', )
```

Comment:
Nit: it's weird to have the trailing comma in a decorator

also going back and forth between keeping this file the same and changing the util to keep pickling for now, but whichever you want to go with seems fine

Reply:
Frankly, if it being an eye-sore means somebody sooner will swap out the serializer for the default json one, then I'm cool with that.

Reply:
I like defaulting the util to json, so new code gets written right the first time.

Reply:
yeah. that makes sense. at first I was suprised that it worked, but I did a quick test locally and it seems fine